src/containers/shared/components/SearchResults/SearchResultRow.tsx

```diff
@@ -0,0 +1,131 @@
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+import Logo from '../../images/no_token_logo.svg'
+import { Amount } from '../Amount'
+import { localizeNumber } from '../../utils'
+import './SearchResultRow.scss'
+import { convertHexToString } from '../../../../rippled/lib/utils'
+import { parsePrice, stripDomain } from './utils'
+
+const renderLogo = (resultContent) =>
+  resultContent.meta.token.icon ? (
+    <object data={resultContent.meta.token.icon} className="result-row-icon">
+      <Logo className="result-row-icon" />
+    </object>
+  ) : (
+    <Logo className="result-row-icon" />
+  )
+
+const renderCurrency = (resultContent) =>
```

Comment:
This conversion is done other places.  There is a Currency component that will handle all of this.